classes/order/OrderReturn.php

```diff
@@ -102,6 +102,9 @@ public function checkEnoughProduct($order_detail_list, $product_qty_list, $custo
         /* Quantity check */
         if ($order_detail_list) {
             foreach (array_keys($order_detail_list) as $key) {
+                if (!isset($product_qty_list[$key])) {
+                    return false;
+                }                 
```

Comment:
```suggestion
                }
```

Reply:
> 



> 
> 
> Hi @Codencode,
> 
> Can use please fix your code to make PrettyCI pass? Calling the script `vendor/bin/php-cs-fixer fix` will help.

Hi @Quetzacoalt91,

I believe I have solved the problem. 
Thanks for the suggestion.